### DISCUSSION TITLE
BuyerRequestedCancel parameter has been added to Orders API

### DISCUSSION THREAD

**Discussion by kelvynf** [abuzuhri/Amazon-SP-API-CSharp#291] [Ideas]
On 3/23 it looks like amzn/selling-partner-api-models Order API was updated to include BuyerRequestedCancel functionality (perhaps there have been other updates as well).  

Can you update the swagger codegen'ed files to reflect these changes.

TIA
Kelvyn

**Comment by kelvynf** [comment]
So I downloaded and built the swagger code generator.  Then used it to produce the c# classes for the Orders api.  Overall the output was very similar but not identical (protected vs public empty constructor, namespace, and different import definitions).  There were the changes to OrderItem and a new BuyerRequestedCancel class that I had expected.  So I've patched my implementation.

I'm relatively new to a lot of the techniques used here but the changes observed in the generated code makes me wonder whether a different swagger utility was used to generate the code, or perhaps some changes were made by hand after the generation?  So I don't know whether to submit the changes to the two files that were modified or not.  If the files are the direct output of a sophisticated, custom configured generation process which the author uses, then adding my changed files from my newbie attempt would just complicate things.

**Comment by abuzuhri** [comment]
its better in this case to compare the different between the versions and update the code depend on the different
